config/routechangerconfig.go

```diff
@@ -6,17 +6,25 @@ import (
 	"strings"
 )
 
-type routeChangerConfig struct {
+type routeChangerConfigItem struct {
 	Reg  *regexp.Regexp
 	Repl string
 	Sep  string
 }
 
+type routeChangerConfig []routeChangerConfigItem
+
 func (rcc routeChangerConfig) String() string {
-	if rcc.Reg == nil {
-		return ""
+	b := strings.Builder{}
```

Comment:
minor: this is equivalent but stdlib example suggests to use `var b strings.Builder`, see https://pkg.go.dev/strings#Builder